style/base.css

```diff
@@ -88,6 +88,11 @@
   display: flex;
   align-items: center;
   gap: 0.5rem;
+  height: 80%;
```

Comment:
why 80%? I prefer to allow the parent to take all the height and set the height of the children (icon, title...) as needed